xbmc/filesystem/MusicDatabaseDirectory.cpp

```diff
@@ -93,6 +93,18 @@ NODE_TYPE CMusicDatabaseDirectory::GetDirectoryParentType(const std::string& str
   return pParentNode->GetChildType();
 }
 
+bool CMusicDatabaseDirectory::GetDirectoryNodeInfo(const std::string& strPath,
+                                                   MUSICDATABASEDIRECTORY::NODE_TYPE& type,
+                                                   MUSICDATABASEDIRECTORY::NODE_TYPE& childtype,
+                                                   MUSICDATABASEDIRECTORY::CQueryParams& params)
+{
+  std::string path = CLegacyPathTranslation::TranslateMusicDbPath(strPath);
+  if (!CDirectoryNode::GetNodeInfo(strPath, type, childtype, params))
```

Comment:
"path" seems unused. This seems not to bet intended.

Reply:
Yes unintended, thanks for spotting. My testing did not using old things like "musicdb://4".

Reply:
Do you think all these year later there are any skins/scripts/keyboard files using the old paths?

Reply:
No idea, sorry.

Reply:
Well anyway it does handle it now as intended (I tested)